react-native-libraries.json

```
@@ -14004,5 +14004,18 @@
     "ios": true,
     "android": true,
     "newArchitecture": true
+  },
+  {
+    "githubUrl": "https://github.com/Kushalchg/nepali-picker",
+    "npmPkg": "react-native-nepali-picker",
+    "examples": [
+      "https://github.com/Kushalchg/nepali-picker/tree/main/example"
+    ],
+    "images": ["https://github.com/Kushalchg/nepali-picker/blob/main/assets/picker.png"],
+    "ios": true,
+    "android": true,
+    "web":true,
+    "expoGo":true,
+    "newArchitecture":true
```

Formatting nit, alternatively you can run `yarn libraries:cleanup` locally.

```suggestion
    "android": true,
    "web": true,
    "expoGo": true,
    "newArchitecture": true
```